packages/react-devtools-extensions/src/contentScripts/backendManager.js

```diff
@@ -134,7 +139,15 @@ function activateBackend(version: string, hook: DevToolsHook) {
     },
   });
 
-  const agent = new Agent(bridge);
+  const agent = new Agent(
+    bridge,
+    getIfReloadedAndProfiling(),
+    onReloadAndProfile,
+  );
+  // Agent read flags successfully, we can count it as successful launch
+  // Clean up flags, so that next reload won't start profiling
+  onReloadAndProfileFlagsReset();
```

Comment:
Is this a safe assumption? Would it be possible that rendering/profiling start async/later somehow?

Reply:
Yes, because these are reload and profile flags. If Backend was reloaded for profiling, then we should've already read them, when Hook is initialized, or when Agent is constructed.

This doesn't affect basic profiling without reloading.